### DISCUSSION TITLE
Going to act 1 farming

### DISCUSSION THREAD

**Discussion by aorta1** [isid0re/SoloLeveling#149] [Q&A]
Got to lvl 30 and act 5 but char keeps going to pit in act one for what seems like gold farm. Makes enough gold to buy pots then goes back never enough for merc or repairs. Is there a way it can farm somewhere else with higher gold drop?

**Comment by aorta1** [comment]
Nvm eventually got enough gold and made its way back to act 5, was stuck in a loop there for a while

**Reply by isid0re** [reply]
I've been doing some data tracking on the script and levelup times for improving performance. And the pits runs for low gold actually dont add much value for the bot during its leveling times because it spends 20 minutes a pop.
